Please update the `ratatui-image` dependency

I'm currently packaging this crate for Debian (see [ITP](https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=1113781)). Currently, Debian ships the `ratatui-image` crate version 8.x: https://tracker.debian.org/pkg/rust-ratatui-image

However, the current code is incompatible with ratatui-image version 8.x due to breaking API changes in the library. When building with ratatui-image = "8", the following compilation errors occur:

```
error[E0404]: expected trait, found struct `StatefulProtocol`
  --> src/ui.rs:27:35
   |
27 | type ImageProtocols = Vec<Box<dyn StatefulProtocol>>;
   |                                   ^^^^^^^^^^^^^^^^ not a trait
```

Could you please consider upgrading the `ratatui-image` to its latest version? That will ensure the crate can be shipped in Debian without an extra patch

--- a/Cargo.toml
+++ b/Cargo.toml
@@ -41,7 +41,7 @@ arboard = "3.3"
 viuer = "0.7"
 image = "0.25"
 zip = "2.0"
-ratatui-image = "1.0"
+ratatui-image = "8.0"
 
 tokio = { version = "1.0", features = ["rt-multi-thread", "macros", "fs"] }
 

--- a/src/ui.rs
+++ b/src/ui.rs
@@ -24,7 +24,7 @@ use std::io;
 use crate::{document::*, Cli};
 use ratatui_image::{picker::Picker, protocol::StatefulProtocol};
 
-type ImageProtocols = Vec<Box<dyn StatefulProtocol>>;
+type ImageProtocols = Vec<StatefulProtocol>;
 
 pub struct App {
     pub document: Document,
@@ -97,19 +97,17 @@ impl App {
     }
 
     fn init_image_support(&mut self) {
-        // Try to initialize picker from termios on Unix, use default on Windows
+        // Try to initialize picker from terminal query on Unix, use font size on Windows
         #[cfg(unix)]
-        let mut picker = if let Ok(p) = Picker::from_termios() {
+        let picker = if let Ok(p) = Picker::from_query_stdio() {
             p
         } else {
             // Fallback to manual font size
-            Picker::new((8, 16))
+            Picker::from_fontsize((8, 16))
         };
 
         #[cfg(not(unix))]
-        let mut picker = Picker::new((8, 16));
-
-        picker.guess_protocol();
+        let picker = Picker::from_fontsize((8, 16));
 
         // Process all images in the document
         for element in &self.document.elements {

